pipeline/metadata/5.3.yaml

```diff
@@ -1229,3 +1229,19 @@ suites:
       - ibmc
       - rgw
       - stage-7
+
+  - name: "test-rgw-ms-with-archive"
+    suite: "suites/pacific/rgw/tier-2_rgw_ms-archive.yaml"
+    global-conf: "conf/pacific/rgw/rgw_multisite.yaml"
```

Comment:
It looks like we are not using `conf/pacific/rgw/rgw_ms_archive.yaml`. If not required, can we remove `conf/pacific/rgw/rgw_ms_archive.yaml` file

Reply:
@udaysk23 thank you, have the correct conf file added